src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const maxCount = 3;
+
+  const sum = (x) => {
+    count++;
+
+    return function(y) {
+      if (count > maxCount && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return x + y;
+    };
+  };
+
+  return sum;
```

Comment:
```suggestion
  return function sum (x) {
    count++;

    return (y) => {
      if (count > maxCount && count % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return x + y;
    };
  };
```

you can immediately return outer function (sum), and in inner function you can use arrow syntax

Reply:
Agree)